front/lib/run_app.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  cloneBaseConfig,
+  DustProdActionRegistry,
+} from "@app/lib/actions_registry";
+import { planForWorkspace, prodAPICredentialsForOwner } from "@app/lib/auth";
+import { DustAPI } from "@app/lib/dust_api";
+import { Workspace } from "@app/lib/models";
+import { WorkspaceType } from "@app/types/user";
+
+export async function runAppOnWorkspaceId(
+  workspaceId: string,
```

Comment:
You should take owner as an argument here, it's available ~everywhere in the code base normally so you should have it handy at your call site?

Reply:
Discussed IRL -> function removed. Building an owner from workspace moved to auth, and this one does not make sense anymore, there's not much more logic to share in Henry and my job